espnet/asr/pytorch_backend/asr.py

```diff
@@ -957,6 +957,12 @@ def recog(args):
     model, train_args = load_trained_model(args.model, training=False)
     assert isinstance(model, ASRInterface)
     model.recog_args = args
+    if args.quantize_asr_model:
+        logging.info("Use quantized asr model for decoding")
+        dtype = getattr(torch, args.quantize_dtype)
+        model = torch.quantization.quantize_dynamic(
+            model, args.quantize_config, dtype=dtype
+        )
```

Comment:
> For information: using quantization with transducer model will currently throw an error because JointNetwork's linear layers inputs during decoding have less than 2 dimensions.

For now, could you please add a check or assert for transducer model please? For example:

```suggestion
    if args.quantize_asr_model:
        assert "transducer" not in mtl_mode, "Quantization of transducer model is not supported yet."

        logging.info("Use quantized asr model for decoding")
        dtype = getattr(torch, args.quantize_dtype)
        model = torch.quantization.quantize_dynamic(
            model, args.quantize_config, dtype=dtype
        )
```

Reply:
thx

Reply:
can you try "args.quantize_config={torch.nn.LSTM}"  for transducer?

Reply:
My apologies for the late reply, I literally forgot. Specifying LSTM will yields the following error in RNN encoder:

> AttributeError: 'LSTM' object has no attribute 'flatten_parameters'

I recall the same issue was reported in pytorch's github in another contexts and it was fixed in later Pytorch version (>1.7.1 I think?). I'll check